src/components/scenes/GuiPluginListScene.tsx

```diff
@@ -105,6 +117,56 @@ class GuiPluginList extends React.PureComponent<Props, State> {
     }
   }
 
+  componentWillUnmount() {
+    this.componentMounted = false
+    if (this.timeoutId != null) clearTimeout(this.timeoutId)
+  }
+
+  setPluginState(plugins: BuySellPlugins): void {
+    if (this.componentMounted && plugins != null) {
+      this.setState({
+        buySellPlugins: {
+          buy: plugins.buy ?? buySellPlugins.buy,
+          sell: plugins.sell ?? buySellPlugins.sell
+        }
+      })
+    }
+  }
+
+  async updatePlugins() {
+    const { disklet } = this.props.account
+    let diskPlugins
+    try {
+      const fileText = await disklet.getText(PLUGIN_LIST_FILE)
```

Comment:
It is not appropriate to store caches in the account synced settings. Please use state.core.disklet, which is a local disklet, not synced.